glean-core/android/src/main/java/mozilla/telemetry/glean/net/BaseUploader.kt

```diff
@@ -36,7 +85,16 @@ class BaseUploader(d: PingUploader) : PingUploader by d {
             val json = JSONObject(data)
             val indented = json.toString(2)
 
-            Log.d(LOG_TAG, "Glean ping to URL: $path\n$indented")
+            if (indented.length + path.length <= MAX_LOG_PAYLOAD_SIZE_BYTES) {
```

Comment:
Do you really need this special case? Wouldn't the function still print only one chunk without this?

Reply:
It would, we would just get the "[Part 1 of 1]" message when it wasn't necessary.  I want to say that I did this more for a performance reason by short-circuiting to keep from having to create the list and do all the other (rather simple) calculations.

In the interest of simplifying things, I can remove this.

Reply:
You could also add a comment here stating the reason why we're "special casing" :) I'd be fine with that :)